.github/workflows/generate_diffs.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
```

Comment:
I assume you're testing. We don't need this job at pull request checks.

Reply:
Yep! Just added in https://github.com/googleapis/google-cloud-java/pull/8442/commits/fa84ab5974d6b88d45457cd243c4212e5ea78454 to run on non-PR `push` events

Reply:
It appears in the checks. Can you fix this?

<img width="959" alt="Screen Shot 2022-09-23 at 4 49 29 PM" src="https://user-images.githubusercontent.com/28604/192054356-3eaf4770-0f52-47d9-b538-6d6d4c2ff7af.png">

Reply:
Hmm, I'm sure if I can prevent it from showing up on the checks. It seems to show up on a `push` event. 

I can skip it from running all together.
![image](https://user-images.githubusercontent.com/6621793/192056270-a359b542-979d-42ef-bfd4-b6a58cb62900.png)

Reply:
Would you explain why you need this line 19 "  pull_request:" ?

Reply:
> I can skip it from running all together.

Yes, please do so. Remove line 19.

Reply:
Ah good catch! I've removed the check.